apps/web/src/hooks/usePoolMutations.ts

```diff
@@ -330,8 +330,8 @@ const usePoolMutations = (returnPath?: string) => {
           toast.success(
             intl.formatMessage({
               defaultMessage:
-                "Success: This pool has been duplicated successfully.",
-              id: "vlyq02",
+                "Success: This process has been duplicated successfully.",
```

Comment:
Why does this message not follow the same format as the other messages and delete and archive?

Reply:
Addressed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9759/commits/5607ab782a99395ae450a468d499b8fb7f3d27d5